client/components/order-packages/index.js

```diff
@@ -0,0 +1,54 @@
+import React, { PropTypes } from 'react';
+import NumberInput from 'components/number-field/number-input';
+import Gridicon from 'components/gridicon';
+import { sanitize } from 'dompurify';
+
+const OrderPackages = ( { packages, updateValue, dimensionUnit, weightUnit, errors } ) => {
+	const renderItemInfo = ( item, itemIndex ) => (
+		<li key={ itemIndex }>
+			<span dangerouslySetInnerHTML={ { __html: sanitize( item.name ) } } />
+			{ 1 < item.quantity ? ' (x' + item.quantity + ')' : null }
+		</li>
+	);
+
+	const renderPackageDimensions = ( pckg ) => {
+		return [ pckg.length, pckg.width, pckg.height ].map( dim => dim + dimensionUnit ).join( ' x ' );
```

Comment:
We should have a space between each dimension and its unit.  E.g. this shows the problem:

<img width="244" alt="screen shot 2016-07-27 at 12 40 51 pm" src="https://cloud.githubusercontent.com/assets/1595739/17189705/9139d4f0-53f7-11e6-8167-1a11eb63054a.png">